sklbench/report/compatibility.py

```diff
@@ -0,0 +1,199 @@
+# ===============================================================================
+# Copyright 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===============================================================================
+
+import math
+
+import numpy as np
+import pandas as pd
+
+from ..utils.logger import logger
+
+
+def transform_results_to_compatible(results: pd.DataFrame):
```

Comment:
This looks like this function should almost just refer to a json, yaml or txt file which parses necessary changes.  May be fragile for future changes in rapids or sklearnex. Thoughts @ethanglaser ?

Reply:
Compatibility hardly can be defined in one static file due to complexity of applied rules changing final results for report generator.

Reply:
can resolve in follow up PR